languoids/tree/sign1238/deaf1237/latv1245/md.ini

```diff
@@ -37,8 +37,8 @@ glottolog =
 	**hh:vld:Mahoney:Estonian-Latvian-Sign**
 
 [classification]
-sub = Low lexicostatistical percentages to other Eastern European sign languages and ASL in **hh:h:Bickford:EEurope(**):40-41 suggest it is independent but the table notably does not include Swedish and/or Finnish sign languages. **hh:v:Mahoney:Estonian-Latvian-Sign(**) is a detailed comparison of Latvian and Estonian Sign but inconclusive regarding a special relationship between the two (since no other languages are included).
+sub = Low lexicostatistical percentages to other Eastern European sign languages and ASL in **hh:h:Bickford:EEurope(**):40-41 suggest it is independent but the table notably does not include Swedish and/or Finnish sign languages. **hh:vld:Mahoney:Estonian-Latvian-Sign(**) is a detailed comparison of Latvian and Estonian Sign but inconclusive regarding a special relationship between the two (since no other languages are included).
```

Comment:
@d97hah You might need to update your subclf data. I had fixed `v:Mahoney:Estonian-Latvian-Sign` -> `vld:Mahoney:Estonian-Latvian-Sign` already for the last PR.